test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -407,6 +414,15 @@ public void run()
         return allArgs;
     }
 
+    protected static void requireAuthentication()
+    {
+        System.setProperty("cassandra.superuser_setup_delay_ms", "-1");
+
+        DatabaseDescriptor.setAuthenticator(new PasswordAuthenticator());
+        DatabaseDescriptor.setRoleManager(new CassandraRoleManager());
+        DatabaseDescriptor.setAuthorizer(new CassandraAuthorizer());
+    }
+
```

Comment:
The changes to `CQLTester` stem from the need for `requireAuthentication`, and have been ported over from DSE.